curvefs/conf/client.conf

```diff
@@ -183,6 +183,8 @@ diskCache.safeRatio=70
 diskCache.threads=5
 # the max size disk cache can use
 diskCache.maxUsableSpaceBytes=107374182400
+# the max files that can cache
+diskCache.maxFileNums=3000000
```

Comment:
Whether this value can be judged according to the type of the current file system, rather than fixed in the configuration file

Reply:
the limit of the file system is too large, so it may cost many memory.  and we cannot get  the limit directly